ideas/310-core-team-proposal.md

```diff
@@ -16,44 +25,34 @@ Desktop is out of scope, as is protocol research. Whitepaper use case implementa
 
 Core needs an accountable, passionate get-shit-done mentality and drive, in contrast to the meandering blame-shifting.
 
+## Additinal contibutors
```

Comment:
typos

Reply:
Also what's role of Rachel/Hester?

Reply:
No idea, just yet, I want to have a kick-off meeting with them, waiting for the answer.

Reply:
added them as POs in this swarm